### ISSUE TITLE
Getting software

### ISSUE THREAD

**Issue by mattharris** [RISC-OS-Community/YouTube#4]
# Background

Create a (series of) video(s) briefly summarising how to use the provided software repositories in RISC OS 5

# Definition of Done

- [ ] A section on how to use !Store
- [ ] A section on how to use !Packman
- [ ] Describe how to add the RISC OS Community repository to PackMan

**Comment by mattharris** [comment]
@pzaino for this one, I want to include the necessary steps to configure PackMan to point at the RISC OS Community repos but would probably want to wait until they're available at riscoscommunity.org - agreed?

**Comment by pzaino** [comment]
Yes, let's wait for packages.riscoscommunity.org being ready, today I am slow, had issues with my router and trying to fix those, then will get back at completing the new packages repository :)

**Comment by mattharris** [comment]
OK, great. Let me know once the repo is ready

**Comment by mattharris** [comment]
@pzaino have you had a chance to work on the repo yet?

**Comment by pzaino** [comment]
@mattharris yes, repo is almost ready. I am about to move the packages now.

Recently there were some discussions on RISC OS Open forum on changing the way a user should add CA Certificates, however yesterday I have seen there is an update of CACertificates on Packman itself, so for now, the process I described in my blog should still be valid.

I guess you can check it when preparing the video and choose which one is best.

**Comment by pzaino** [comment]
P.S. Sorry for the delay, but I had issues with my home server, so had to fix that as priority.

**Comment by mattharris** [comment]
No worries about the delay - just wanted to be sure that I'd not missed any messages about this.

**Comment by pzaino** [comment]
@mattharris ,
quick update, all packages are migrated, I am still having some issues with nginx server when using the packages.riscoscommunity.org host name, hopefully will be sorted by the end of the week.

**Comment by pzaino** [comment]
@mattharris ,
sorry for the really long delay, got busy on other priorities, anyway, the repo is up and running:

Released software:
https://riscoscommunity.org/packages/rpkg/arm32/released

Beta testing software:
https://riscoscommunity.org/packages/rpkg/arm32/testing

I should start testing the repository this weekend, JFYI.

Configuration instructions are identical to configure my own RiscPkg repository, instructions here:

https://paolozaino.wordpress.com/my-public-software-repositories/#RISCOS-RiscPkg-Repos

As always, thanks for your help Matt :)
